.gitignore

```diff
@@ -478,3 +478,4 @@ target/
 
 # MSVC Windows builds of rustc generate these, which store debugging information
 *.pdb
+src/wasi_samples/wit/component-world.wasm
```

Comment:
should we just add *.wasm (or maybe src/wasi_samples/**/*.wasm?) so we don't need to edit this as we add more examples?

Reply:
done